packages/cli/src/collect/collect.js

```diff
@@ -97,15 +102,19 @@ async function startServerAndDetermineUrls(options) {
     if (options.startServerCommand) {
       const {child, patternMatch, stdout, stderr} = await runCommandAndWaitForPattern(
         options.startServerCommand,
-        /(listen|ready)/i,
+        new RegExp(options.startServerReadyPattern, 'i'),
         {timeout: 10000}
       );
       process.stdout.write(`Started a web server with "${options.startServerCommand}"...\n`);
       close = () => killProcessTree(child.pid);
 
       if (!patternMatch) {
         process.stdout.write(`WARNING: Timed out waiting for the server to start listening.\n`);
-        process.stdout.write(`         Ensure the server prints "listening" when it is ready.\n`);
+        process.stdout.write(
```

Comment:
this got a tad hard to tell why the indent is there :)

how about `const pattern = new RegExp(options.startServerReadyPattern, 'i')` at the top of this block and then

```suggestion
        const msg = `Ensure the server prints text matching ${pattern} when it's ready.\n`;
        process.stdout.write(`         ${message}`);
```

?

Reply:
Sure, but do you want to print the whole regex pattern or just the pattern? The indentation was done by prettier because the line was too long after I added `options.startServerReadyPattern`.

The whole regex pattern would look something like this: `/listen|ready/i` and just the pattern `listen|ready`.

Reply:
regex.source works for me too, though I don't have a strong preference 🤷‍♂  

I thought I recognized the prettier autoflow 😉 I was trying to think of something that preserves just the indentation piece on one line to make it obvious with the previous `process.stdout.write`

Reply:
I went ahead with the full regex option. Maybe it can help someone debugging a start command not working 🤷🏼‍♂️.